Gems/Blast/Code/Source/Components/BlastSystemComponent.h

```diff
@@ -100,11 +100,14 @@ namespace Blast
         class AZBlastAllocatorCallback : public Nv::Blast::AllocatorCallback
         {
         public:
+            // Blast requires 16-byte alignment
+            static const size_t alignment = 16;
```

Comment:
uber nit: (can leave but just mentioning) Should this be `Alignment` as opposed to `alignment`?

Reply:
Yeah... it's ready to merge... I'll keep it in mind for future PRs.